tensorflow/core/ops/nn_ops.cc

```diff
@@ -1999,7 +1999,7 @@ REGISTER_OP("_MklLeakyRelu")
     .Input("mkl_features: uint8")
     .Output("activations: T")
     .Output("mkl_activations: uint8")
-    .Attr("T: {half, float, double} = DT_FLOAT")
+    .Attr("T: realnumbertype = DT_FLOAT")
```

Comment:
`realnumbertype` supports integrals. Do you want to allow int as well?
https://github.com/tensorflow/tensorflow/blob/526f6337bbbaee4492156d358b9d5e90d1c87382/tensorflow/core/framework/op_def_builder.cc#L135-L140
https://github.com/tensorflow/tensorflow/blob/526f6337bbbaee4492156d358b9d5e90d1c87382/tensorflow/core/framework/types.h#L274-L279

Reply:
Good point. Fixed.